changelog/7126.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Switched to ``saferepr`` to format bytes ``parametrize`` parameters
```

Comment:
`saferepr` is an internal name, so people reading this are unlikely to know what it means. I suggest rephrasing to only describe the bug that is fixed, leaving out the actual solution.